front-end/src/components/AnalyzersAndSeries.js

```diff
@@ -3,22 +3,35 @@ import Help from './Help';
 import analyzerData from './analyzerData';
 import '../styles/analyzersAndSeriesStyles.css';
 import { useState, useRef } from 'react';
-const AnalyzersAndSeries = ({ dimensions, columns, setDisplayPage, setShowModal, handleSubmit, setSuccessMessage }) => {
+const AnalyzersAndSeries = ({ dimensions, columns, setDisplayPage, setShowModal, handleSubmit, setSuccessMessage, setDimensions, graphType }) => {
 
     const columnGenerator = (n) => {
         let arr = [];
-        for (let i = 0; i < n; i++) {
+        for (let i = 0; i < 2; i++) {
             arr.push(
-                <div >
-                    <div className="boldText">{i === 0 ? "X-Axis" : "Y-Axis"} </div>
-                    <select className={i} key={i}>
+                <div>
+                    <div className="boldText">{i === 0 ? "X-Axis" : "Y Axis"} </div>
+                    <select className={i} key={i} defaultValue={JSON.stringify(columns[i])}>
                         {columns.map(column => (
                             <option value={JSON.stringify(column)} key={column.header + column.filename}>{column.filename} - {column.header}</option>
                         ))}
                     </select>
                 </div>);
         }
-        return arr;
+        let arr2 = [];
```

Comment:
We're going to need to fix this code, but since we will be redoing this anyways, it can stay for now